### PR TITLE
[14.0] [ADD] `sale_order_invoice_date` : Display the invoice date on Sales Order analysis reports

### PR DESCRIPTION
**Pull request by ivantodorovich**
Display the invoice date on Sales Order analysis reports

### CODE REVIEW

File: sale_order_invoice_date/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Display the invoice date on Sales Order analysis reports.
```
Comment (rousseldenis) [inline sale_order_invoice_date/readme/DESCRIPTION.rst:1]: @ivantodorovich Could you just add a line that indicates user that invoice date is the last one if several invoices for the same sale order ?

**Review by jcoux — APPROVED** [review]

File: sale_order_invoice_date/readme/DESCRIPTION.rst (hunk 1 above)
Comment (ivantodorovich) [reply sale_order_invoice_date/readme/DESCRIPTION.rst:1]: Done 👌🏻 thanks!!

File: sale_order_invoice_date/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+The invoice date is considered to be the latests invoice date linked to fully
```
Comment (rousseldenis) [inline sale_order_invoice_date/readme/DESCRIPTION.rst]:
```suggestion
The invoice date is considered to be the latest invoice date linked to fully
```

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**